semgrep_output_v1.atd

```diff
@@ -1282,7 +1291,11 @@ type finding = {
   end_column: int;
 
   message: string;
-  (* int|string until minimum version exceeds 1.32.0, then string *)
+  (* int|string until minimum version exceeds 1.32.0, then int?
+   * Error = 2, Warning = 1, Experiment = 4, otherwise 0
```

Comment:
Above it says deprecating Experiment but here it looks like it still has its own int?

Reply:
I'm just commenting here what pysemgrep is currently doing. I'll be happy to remove that.
But really I would be happier if we were switching from an int to a string (actually an enum, that is the one in match_severity) actually going forward.